### ISSUE TITLE
Questions regarding the portrayal of ferry cable

### ISSUE THREAD

**Issue by DavidGrant-NIWC** [iho-ohi/S-101_Portrayal-Catalogue#443]
`CableSubmarine` with `categoryOfCable`=`7 - Ferry` should use viewing group `34070` vice `24010`:

![image](https://github.com/user-attachments/assets/a2130c40-cc37-428c-bc5c-18c8d8311ec6)

![image](https://github.com/user-attachments/assets/70bb0038-4889-4765-b524-c186f5a28f65)

related: #59

**Comment by DavidGrant-NIWC** [comment]
@alvarosanuy do you agree with this change (from row 3 to row 4)? If not, then the description of viewing group 24010 should be updated to include ferry cables. Also, it seems odd that ferry cables use `CHMGD` but don't alert whereas junction cables will use `CHMGF` and do alert; I guess it's too late to address that in this version.

|ViewingGroup|DrawingPriority|LineStyle|Remarks|
|:---:|:---:|:---|:---|
|34070|3 (equivalent to 9 in S-101)|CBLSUB06|S-52|
|34070|9|CBLSUB06|S-101 prior to #59|
|24010|18|CHMGD thin dashed|S-101 post #59|
|34070|9|CHMGD thin dashed|S-101 change proposed by this issue|

![image](https://github.com/user-attachments/assets/2e8fd9f3-c9c6-4015-b69f-8ab9b99dceed)

**Comment by alvarosanuy** [comment]
I did a bit of research, and I found that Cable Ferry Routes and Areas are currently portrayed using CHBLK. This is probably due to the fact that a cable ferry is considered a danger to navigation due to:

- their restricted maneuverability and
- the use of cables/chains that can be suspended from the bottom to a certain distance of the Feery and therefore reduce nearby natural depths.

Accordingly, to me, the new **categoryOfCable**=7 (ferry) should be considered a potential navigation hazard and therefore portrayed using the same colour than cable ferry routes and areas (CHBLK or CHGRD?). Magenta should be used for cables and pipelines that sit on the seafloor.

**In summary, I recommend: **

1. (CBLSUB, CBLSUB=7) is added to VG 24010
2. Cable Ferries should trigger an indication. I understand it wouldn't affect S-52 as this type of CBLSUB does not exist in S-57.

**Comment by DavidGrant-NIWC** [comment]
@TomRichardson6, @alvarosanuy 

Note:
* Alvaro's summary doesn't propose changing the color (to `CHBLK` or `CHGRD`, currently `CHMGD`) but the description does.
* Items which generate alerts are usually included in a viewing group in the `14000` range:
  * The description of `14010` was recently updated to include junction cables (which now generates an alert).<br/> ![image](https://github.com/user-attachments/assets/339bef1c-a17d-4ec5-9887-c31eaa9bf918)
  * The description of `24010` was recently updated to remove junction cables. The remaining items (including mooring cables) do not (currently) generate alerts.<br/>
![image](https://github.com/user-attachments/assets/b016c500-3453-40a4-87cf-ff6e6e1318b7)

### Changes for Ferry Cables (`CableSubmarine` with `categoryOfCable`=`7 - Ferry`)
1. Associate a `NavHazard` alert with all instances of ferry cables?
    - [ ] Yes
    - [ ] No
3. Which viewing group?
    - [ ] `14010` and update description to include ferry cables (recommended if answer to 1 is yes)
    - [ ]  (Continue to use) `24010` and update description to include ferry cables (recommended if answer to 1 is no)
4. Which color?
    - [ ] (Continue to use) `CHMGD`
    - [ ] `CHBLK`
    - [ ] `CHGRD`

**Comment by TomRichardson6** [comment]
My view is that more discussion is needed (for change in a future S-101 version if required) here before making a change that adds alerts for this feature as they do not alert in S52. So my responses are as follows;

1. No

2. Continue to use 24010

3. Continue to use CHMGD

In my experience it is common that the Ferry Route is charter but the specific cables are not. For the instances I am aware of the cables only pose a hazard in proximity to the ferry itself where they are raised from the seabed. Given the significant feedback on alerts in ECDIS a full discussion is needed before adding new alerts.
